lib/tasks/gather.js

```diff
@@ -162,8 +162,9 @@ class TaskGather extends Task {
         alert_type: AlertType.STT_NOT_PROVISIONED,
         vendor: this.vendor
       }).catch((err) => this.logger.info({err}, 'Error generating alert for no stt'));
-
-      throw new Error(`no speech-to-text service credentials for ${this.vendor} have been configured`);
```

Comment:
I was thinking we still wanted to throw the error, just that before doing so we should send the jambonz:error message to the listening websocket client.

Reply:
fixed